Justfile

```diff
@@ -752,11 +752,9 @@ generate-build-tags image="bluefin" tag="latest" flavor="main" kernel_pin="" ghc
     DEFAULT_TAG=$(just generate-default-tag {{ tag }} {{ ghcr }})
     IMAGE_NAME=$(just image_name {{ image }} {{ tag }} {{ flavor }})
     # Use Build Version from Rechunk
-    BUILD_VERSION={{ version }}
-    if [[ -z "${BUILD_VERSION:-}" ]]; then
-        BUILD_VERSION="${FEDORA_VERSION}.$(date +%Y%m%d)"
+    if [[ -z "${version:-}" ]]; then
+        version="${FEDORA_VERSION}.$(date +%Y%m%d)"
     fi
-    BUILD_VERSION="${BUILD_VERSION:3}"
```

Comment:
This means that our dated tags will now be of the style
<stream>-<fedora_version>.<date>

and no longer just <stream>-<date>

This is to work with rechunk's ability to update the version.

Reply:
this is back in.